### ISSUE TITLE
Paroxysmal Tonic Upgaze

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7040]
**Preferred term label:**
Paroxysmal tonic upgaze

**Synonyms**

**Definition (free text, please give PubMed ID)**
Recurrent episodes of sustained upward deviation of the eyes and incomplete downward saccades, and normal horizontal eye movements without impairment of consciousness.
PMID: 29095389

**Parent term (use hpo.jax.org/app)**
Abnormality of eye movement HP:0000496

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:147750   3/4  PMID: 33783914

Paroxysmal Tonic Upgaze, Benign Childhood, With Ataxia OMIM:168885

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
